spark/src/test/scala/io/delta/tables/DeltaTableBuilderSuite.scala

```diff
@@ -18,18 +18,22 @@ package io.delta.tables
 
 // scalastyle:off import.ordering.noEmptyLine
 import org.apache.spark.sql.delta.DeltaLog
+import org.apache.spark.sql.delta.skipping.ClusteredTableTestUtils
 import org.apache.spark.sql.delta.sources.DeltaSourceUtils.GENERATION_EXPRESSION_METADATA_KEY
 import org.apache.spark.sql.delta.sources.DeltaSQLConf
 import org.apache.spark.sql.delta.test.DeltaSQLCommandTest
-import org.apache.spark.sql.delta.test.DeltaTestImplicits._
 
 import org.apache.spark.sql.{AnalysisException, QueryTest}
 import org.apache.spark.sql.catalyst.TableIdentifier
-import org.apache.spark.sql.catalyst.analysis.{CannotReplaceMissingTableException, NoSuchDatabaseException, TableAlreadyExistsException}
+import org.apache.spark.sql.catalyst.analysis.TableAlreadyExistsException
 import org.apache.spark.sql.test.SharedSparkSession
 import org.apache.spark.sql.types.{IntegerType, LongType, MetadataBuilder, StringType, StructType}
 
-class DeltaTableBuilderSuite extends QueryTest with SharedSparkSession with DeltaSQLCommandTest
+class DeltaTableBuilderSuite
+    extends QueryTest
+    with SharedSparkSession
+    with DeltaSQLCommandTest
+    with ClusteredTableTestUtils
```

Comment:
We use 2 space indentation: https://github.com/databricks/scala-style-guide?tab=readme-ov-file#spacing-and-indentation
```suggestion
  extends QueryTest
  with SharedSparkSession
  with DeltaSQLCommandTest
  with ClusteredTableTestUtils
```

Reply:
Updated